src/main/resources/db/migration/0200/V200__BackfillPopulations.sql

```diff
@@ -0,0 +1,50 @@
+INSERT INTO tracking.planting_subzone_populations (
+    planting_subzone_id,
+    species_id,
+    total_plants,
+    plants_since_last_observation)
+SELECT plantings.planting_subzone_id,
+       plantings.species_id,
+       SUM(plantings.num_plants),
+       SUM(plantings.num_plants)
+FROM tracking.plantings
+WHERE plantings.planting_subzone_id IS NOT NULL
+GROUP BY plantings.planting_site_id, plantings.planting_subzone_id, plantings.species_id
+ON CONFLICT (planting_subzone_id, species_id)
```

Comment:
conflicts likely in the migration or just in case we need to re-run?

Reply:
If there was an outplanting withdrawal in between the time the previous code change was released and this one is released, these tables will already have some data in them, but the data will be wrong (because the code assumes the numbers from previous plantings are already reflected in these tables) so we need to overwrite it in the migration.

Reply:
Thanks, makes sense. Yes, I forgot we will be running this migration after the BE has been upgraded with the other changes (handling plants since last observation).